tests/test_searcher.py

```diff
@@ -14,6 +16,14 @@
 from sporkfish.searcher.searcher_factory import SearcherFactory
 
 
+def _evaluator(
```

Comment:
Naive python style q - I thought a free function with _ in front of it would not be imported to other files? see [this](https://stackoverflow.com/questions/8689964/why-do-some-functions-have-underscores-before-and-after-the-function-name) and/or [this](https://peps.python.org/pep-0008/#descriptive-naming-styles).

From a style perspective, it looks to me as a suggestion that programmer's intent is to only to be used in this current module/file (kind of like [anonymous namespace in C++](https://stackoverflow.com/questions/5238932/namespace-without-a-name-in-c)).

Thoughts??

Reply:
Thats fair, which was why I ended up defining it twice. Should I move it to init_board_helper instead?

Reply:
That's good shout actually - think would be neater.

@KYLChiu ?

Reply:
No objections!